server/db/schema.ts

```diff
@@ -20,8 +20,7 @@ import { type AdapterAccount } from "next-auth/adapters";
 export const role = pgEnum("Role", ["MODERATOR", "ADMIN", "USER"]);
 
 export const session = pgTable("session", {
-  id: text("sessionToken").primaryKey(),
-  sessionToken: text("sessionToken").notNull(),
+  sessionToken: text("sessionToken").notNull().primaryKey(),
```

Comment:
I dont like this and will need to test further with the rest of the app but it wouldnt let with interact with the session table until I resolved this duplicate table reference

Reply:
So why do we have to delete the ID? 🤔

Reply:
Because its actually wrong. If we look at the table we can see that it only has three columns. id isnt there

<img width="778" alt="image" src="https://github.com/user-attachments/assets/9bc118a0-aea1-4a38-b7e2-1d3010d2ee83">